lib/ansible/modules/cloud/amazon/ec2_vpc_nacl_info.py

```diff
@@ -132,7 +136,7 @@ def list_ec2_vpc_nacls(connection, module):
         nacl_ids = []
 
     try:
-        nacls = connection.describe_network_acls(NetworkAclIds=nacl_ids, Filters=filters)
```

Comment:
You could just leave this here and remove _describe_network_acls. To retry you'll need to pass in aws_retry=True which defaults to False for safety:
```python
nacls = connection.describe_network_acls(NetworkAclIds=nacl_ids, Filters=filters, aws_retry=True)
```